core-bundle/src/Resources/contao/forms/Form.php

```diff
@@ -405,18 +405,18 @@ protected function processFormData($arrSubmitted, $arrLabels, $arrFields)
 			$email->fromName = $GLOBALS['TL_ADMIN_NAME'];
 
 			// Get the "reply to" address
-			if (!empty(Input::post('email', true)))
+			if (!empty($arrSubmitted['email']))
 			{
-				$replyTo = Input::post('email', true);
+				$replyTo = StringUtil::decodeEntities($arrSubmitted['email']);
```

Comment:
This should not be necessary because `decodeEntities` is enabled for all widgets by default:

https://github.com/contao/contao/blob/3281624a9095b467a85fe8a012d601a9df24d009/core-bundle/src/Resources/contao/forms/Form.php#L166

Reply:
Indeed (tested it as well). Last commit reverted.